src/components/InventoryHostStaleness/constants.js

```diff
@@ -405,15 +405,55 @@ export const systemStalenessItems = (activeTabKey) => {
 export const systemStalenessWarningItems = (activeTabKey) => {
   return [
```

Comment:
I think we can do these nicer. Maybe an array 

```js
const allDays = [2, 3, 4, 5, 6, 7, 14, 21, 30, 60, 90, 120];
const apiKey = activeTabKey ? 'immutable_time_to_stale_warning' : 'conventional_time_to_stale_warning';
const title = 'System stale warning';

allDays.map((value) => ({
       name: value + ' days',
       value,
       apiKey,
       title,
     })
```


Same below. :)

Reply:
ahh nice implementation. cuts down the file size a bit, sendin this up!